src/sentry/workflow_engine/handlers/action/notification/issue_alert.py

```
@@ -26,14 +30,46 @@ class BaseIssueAlertHandler(ABC):
     """
 
     @staticmethod
+    def get_integration_id(action: Action, mapping: ActionFieldMapping) -> dict[str, Any]:
+        if mapping.get("integration_id_key"):
+            return {mapping["integration_id_key"]: action.integration_id}
+        return {}
+
+    @classmethod
+    def get_target_identifier(cls, action: Action, mapping: ActionFieldMapping) -> dict[str, Any]:
+        if mapping.get("target_identifier_key"):
+            return {mapping["target_identifier_key"]: action.target_identifier}
+        return {}
```

Does it make sense to have this fallback to an empty value? It may make more sense for us to raise if there's a missing mapping, forcing classes without mappings to explicitly override these classmethods instead in order to prove they're handling these cases explicitly.

---

yep agreed, now this should raise an exception.